atlasdb-tests-shared/src/main/java/com/palantir/atlasdb/sweep/AbstractSweeperTest.java

```diff
@@ -501,11 +531,23 @@ public void testBackgroundSweepCanHandleNegativeImmutableTimestamp() {
         });
 
         for (SweepPriorityRowResult result : results) {
-            Assert.assertEquals(new Long(Long.MIN_VALUE), result.getMinimumSweptTimestamp());
+            Assert.assertEquals(Long.valueOf(Long.MIN_VALUE), result.getMinimumSweptTimestamp());
         }
     }
 
     @Test
+    public void testBackgroundSweeperSendsMetrics() {
```

Comment:
Maybe also add a test that metric reporting is as expected when we have more rows than row batch size?